contrib/src/main/java/macrobase/analysis/stats/kde/QuantileBoundEstimator.java

```diff
@@ -0,0 +1,160 @@
+package macrobase.analysis.stats.kde;
+
+import macrobase.analysis.stats.kde.kernel.BandwidthSelector;
+import macrobase.analysis.stats.kde.kernel.Kernel;
+import macrobase.analysis.stats.kde.kernel.KernelFactory;
+import org.apache.commons.math3.stat.descriptive.rank.Percentile;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+public class QuantileBoundEstimator {
+    private static final Logger log = LoggerFactory.getLogger(QuantileBoundEstimator.class);
+
+    public TKDEConf tConf;
```

Comment:
hmm, 

1.) why do we need a separate conf object?

2.) if this is TKDE specific, why not inner class?